spec/helpers/holding_block_spec.rb

```diff
@@ -240,6 +240,24 @@
       end
     end
 
+    context '#holding_block_search with links only' do
+      let(:document) do
+        {
+          id: '1',
+          format: ['Book'],
+          electronic_access_1display: '{"https://library.princeton.edu/resource/28076":["library.princeton.edu"]}'
+        }.with_indifferent_access
+      end
+
+      before { stub_holding_locations }
+
```

Comment:
Can you update this with the `stub_alma_holding_locations` https://github.com/pulibrary/orangelight/blob/57511ef6189e437fd311f2ad0f5ed2c8fb4649de/spec/support/webmock_stubs.rb#L9 ?

Reply:
All the existing tests in this file use `stub_holding_locations`. If we want to refactor the code and leave only one stub method (either `stub_holding_locations` or `stub_alma_holding_locations`) I would suggest we handle that via a separate PR.

Reply:
That's fine. It can be done at another time. I have already updated in some places to use the stub_alma_holding_locations.